testsuite/examples/test_framework_GRASS_GIS_with_NC.conf

```diff
@@ -27,3 +27,15 @@ REPORTS="testreports"
 PUBLISH="no"
 # upload WWW dir on server for report publication (not used for local tests)
 SERVERDIR="/var/www/html/grassgistestreports"
+
+# read module tests list from file
+# may be no|yes
+READ_MODULE_TEST_LIST=yes
+# file with list of tests to be run
+TEST_LIST_IN=/src/grass_build/testsuite/examples/test.list
```

Comment:
I would prefer if paths were not hardcoded. See also #5

Reply:
changed this one as well